pkg/collector/corechecks/snmp/internal/report/report_metrics.go

```diff
@@ -122,9 +122,16 @@ func (ms *MetricSender) reportColumnMetrics(metricConfig checkconfig.MetricsConf
 	rowTagsCache := make(map[string][]string)
 	samples := map[string]map[string]MetricSample{}
 	for _, symbol := range metricConfig.Symbols {
-		metricValues, err := getColumnValueFromSymbol(values, symbol)
-		if err != nil {
-			continue
+		var metricValues map[string]valuestore.ResultValue
+
+		if symbol.ConstantValueOne {
+			metricValues = getConstantMetricValues(metricConfig.MetricTags, values)
+		} else {
+			var err error
+			metricValues, err = getColumnValueFromSymbol(values, symbol)
```

Comment:
```suggestion
			metricValues, err := getColumnValueFromSymbol(values, symbol)
```
🥜 shorter syntax?

Reply:
`metricValues` is declared line 125 (outside of the if statement) so using `:=` gives `Unused variable 'metricValues'`, I don't know how to better write this 🙁

Reply:
ah yes, it's fine then :)